STW-Files/ESP-Files/aob-kulfan.csm

```diff
@@ -0,0 +1,235 @@
+# Aeroelastic Optimization Benchmark model geometry script
+# Written by Brian J. Burke
+
+cfgpmtr view:flow 1
+cfgpmtr view:struct 0
+
+set mirror 0
+# I have not built support for mirroring when using Kulfan.
+
+cfgpmtr nribs 23
+cfgpmtr nspars 2
+
+set cfd:wakeSheet 0
+set cfd:sharpTE 0   
+# cfd:sharpTE doesn't do anything, just keeping it here so I can pass it to 
+# not break other stuff. Finite TE is now set through the ztail parameters.
+
+cfgpmtr xFarfield 500
+
+## DESIGN PARAMETERS
+# ---------------------------------
+# General wing and airfoil parameters
+despmtr rootChord 5.0   # Root chord length
+despmtr tipChord 1.5    # Tip chord length
+despmtr halfSpan 14.0   # Half span length (y direction)
+despmtr tipOffset 7.5   # Tip offset in x direction
+
+## Kulfan airfoil parameters and geometric twist
+# Five stations plus the root are prescribed to determine the wing shape.
+# At each station, four coefficients are used for the upper and lower surface
+# each for the Kulfan airfoil.
+# *-------- KULFAN PARAMETERS --------*
+# Root (not active in the design, set as configuration parameters)
```

Comment:
FYI, the root airfoil shape is allowed to change, it's just the twist that needs to be fixed